### PR TITLE
Add shift-add and shift-delete commands and make commands case-insensitive

### LINKED ISSUE #10: Add shifts
As a user I can add new shifts so I can assign workers to those shifts

### LINKED ISSUE #12: Add positions to shifts
As a user I can add positions (eg. cashier, cleaner, etc.) that need to be filled in each shift so that I can assign workers to those positions

### LINKED ISSUE #15: Delete shifts
As a user I can delete a shift so that I can remove unwanted shifts

### PR DESCRIPTION
**Pull request by plosslaw**
Resolves #10 
Resolves #12 
Resolves #15 

Test cases will be added in future commits

### CODE REVIEW

**Comment by plosslaw** [comment]
Currently, the shift-add and shift-edit commands do not support role requirements that uses spaces (e.g. Fry Cook) #44

File: docs/UserGuide.md [hunk 1]
```diff
@@ -173,8 +173,8 @@ Format: `shift edit SHIFT_INDEX [d/DAY] [t/TIME] [r/ROLE-NUMBER_NEEDED]...`
 * Each role should be accompanied by the number needed, and this number **must be 0 (0 will remove the role from the shift) or a positive integer** 1, 2, 3...
```
Comment (WangZijun97) [inline docs/UserGuide.md:173]: Just tested with the current `master` branch code, 0 does not remove the role from the shift yet, but instead shift-edit removes all roles, then add the ones keyed in. Do we want to have the 0 = remove functionality, or just not allow 0 to be keyed in?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -163,7 +163,7 @@ Format: `shift list`
 
 Edits the details of an existing shift in the McScheduler.
 
-Format: `shift edit SHIFT_INDEX [d/DAY] [t/TIME] [r/ROLE-NUMBER_NEEDED]...`
+Format: `shift edit SHIFT_INDEX [d/DAY] [t/TIME] [r/ROLE <SPACE> NUMBER_NEEDED]...`
```
Comment (WangZijun97) [inline docs/UserGuide.md:166]: Thanks for helping to update the user guide.

**Review by WangZijun97 — APPROVED** [review]
LGTM! Good fix on the lower case commands. Added a question, but approving as that comment isn't really about the code you are pushing.

File: docs/UserGuide.md (hunk 1 above)
Comment (plosslaw) [reply docs/UserGuide.md:173]: I think we'll still go with 0 removing the role

**Pull request merged**